client/src/pages/HomePage.jsx

```diff
@@ -16,7 +16,7 @@ const HomePage = (props) => {
         <HomePageContent />
         <Grid container justifyContent="center">
           <Grid item lg={8} md={9} xs={11}>
-            <SearchBar />
+            <SearchBar isHomePage />
```

Comment:
instead of `isHomePage` maybe we can use a prop name that is more fitting to what we're doing maybe like `narrow` or `wide` or something that describes this. cause out of the context of the page, i think the prop should tell us what it does